test/conformance/helpers/broker_data_plane_test_helper.go

```diff
@@ -72,6 +81,7 @@ func BrokerV1Beta1IngressDataPlaneTestHelper(
 	trigger := client.CreateTriggerOrFailV1Beta1(
 		triggerName,
 		resources.WithBrokerV1Beta1(broker.Name),
+		resources.WithNamespaceTriggerV1Beta1(broker.Namespace),
```

Comment:
We don't need to set the Trigger namespace since `CreateTriggerOrFailV1Beta1` uses `client.Namespace`.

Reply:
ah forgot to remove these, doh, thanks